pkg/ebpf/probes/trace.go

```diff
@@ -77,6 +96,19 @@ func (p *TraceProbe) attach(module *bpf.Module, args ...interface{}) error {
 		return errfmt.WrapError(err)
 	}
 
+	var ksyms *environment.KernelSymbolTable
```

Comment:
I think we may want to move this code section inside the switch case in the kprobe case. Otherwise, other traceprobes which don't need this will require it for attachment.